### PR TITLE
WNPS Strong motion sensor swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removed SCI CSUP3D (42040) from location 20 HN-12Z
- Replaced with CSI EQR120 at same location
- Firmware version 2.2.2, @CBurton90  is this right?
- This station has been continous since 2020-02-09T21:50:00

### CODE REVIEW

**Review by salichon — APPROVED** [review]

**Comment by CBurton90** [comment]
@Thomas-Benson Firmware is `2.3.2` (should be that for all new EQR120's going out)

File: install/firmware.csv [hunk 1]
```diff
@@ -297,6 +297,7 @@ Canterbury Seismic Instruments,EQR120,46498,m001.03.06,2021-03-31T23:09:00Z,2021
 Canterbury Seismic Instruments,EQR120,46498,m001.03.08,2021-04-13T21:29:44Z,2021-04-28T21:44:00Z,
 Canterbury Seismic Instruments,EQR120,46499,m001.03.06,2021-02-11T02:49:00Z,2021-04-13T22:01:48Z,
 Canterbury Seismic Instruments,EQR120,46499,m001.03.08,2021-04-13T22:01:49Z,2021-04-28T02:19:00Z,
+Canterbury Seismic Instruments,EQR120,46499,2.2.2,2023-06-21T00:01:00Z,9999-01-01T00:00:00Z,
```
Comment (CBurton90) [inline install/firmware.csv:300]: 2.3.2

**Review by CBurton90 — CHANGES_REQUESTED** [review]
Looks good after fw correction : )

File: install/firmware.csv (hunk 1 above)
Comment (Thomas-Benson) [reply install/firmware.csv]: fixed firmware

**Review by CBurton90 — APPROVED** [review]

**Review by mnaguit — APPROVED** [review]

**Pull request merged**